src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,569 @@
+providing media to MSE.
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=2 style="white-space: nowrap">
+<a href="#link_preload">Link preload</a>
+      </td>
+      <td>
+Forces the browser to make a request for a video resource without blocking
+the document's <code>onload</code> event.
+      </td>
+      <td>
+HTTP Range requests are not compatible.
+      </td>
+    <tr>
+      <td>
+Compatible with MSE and file segments.
+      </td>
+      <td>
+Should be used only for small media files (<5 MB) when fetching full resources.
+      </td>
+    </tr>
+    <tr>
+      <td>
+<a href="#manual_buffering">Manual buffering</a>
+      </td>
+      <td>
+Full control
+      </td>
+      <td>
+Complex error handling is the website's responsibility.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Video preload attribute
+
+If the video source is a unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when the initial HTML document has been
+completely loaded and parsed (e.g. the `DOMContentLoaded` event has fired)
+while the very different `window.onload` event will be fired when resource
+has actually been fetched.
+
+<figure>
+  <img src="/web/fundamentals/media/images/video-preload/video-preload.svg">
+</figure>
+
+Setting the `preload` attribute to `metadata` indicates that the user is not
+expected to need the video, but that fetching its metadata (dimensions, track
+list, duration, and so on) is desirable.
+
+```
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('loadedmetadata', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+Setting the `preload` attribute to `auto` indicates that the browser may 
+cache enough such that the playback to end would be possible without 
+requiring a stop for further buffering.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('loadedmetadata', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, the browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver] is enabled, Chrome forces the `preload` value to `none`.
+- In Android 4.3, Chrome forces the `preload` value to `none` due to an [Android bug].
```

Comment:
Webview that shipped in Android 4.3 doesn't have this enhancement. So it's only in Chrome.